test/scripts/clusters/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
```suggestion
set -e -o pipefail
```

Otherwise it won't actually stop the test in case it fails, and will exit successfully eventually.

Reply:
Added